cmd/sealer/cmd/alpha/prune.go

```diff
@@ -0,0 +1,74 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/sealerio/sealer/pkg/prune"
+
+	"github.com/spf13/cobra"
+)
+
+var exampleForPruneCmd = `The following command will prune sealer data directory such as, image db ,image layer, build tmp.
```

Comment:
`The following command will prune sealer data directory, such as: image db, image layer and build tmp.`